adobe/react-spectrum
# #2610: update useMenuTrigger.mdx to pass menuProps directly to MenuPo…

This change updates useMenuTrigger.mdx to resolve warning aria related errors. 

Closes 
#2610 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. Open the useMenuTrigger documentation page: http://localhost:1234/react-aria/useMenuTrigger.html
2. Scroll down to the example
3. Open dev tools console
4. Click the example to open the menu dropdown. Confirm that no warnings are sent to console.

## 🧢 Your Project:
N/A
